src/main/java/Deadline.java

```diff
@@ -0,0 +1,23 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task {
+    String dead;
+    LocalDate deadDate;
+    public Deadline(String details, String dead) {
```

Comment:
The parameter name dead is a bit confusing, considering it has the meaning of death. Perhaps a more intuitive name here?